pkg/controller/certificates/certificate_controller.go

```diff
@@ -151,13 +165,13 @@ func (cc *CertificateController) enqueueCertificateRequest(obj interface{}) {
 		utilruntime.HandleError(fmt.Errorf("Couldn't get key for object %+v: %v", obj, err))
 		return
 	}
-	cc.queue.Add(key)
+	cc.queue.AddAfter(key, cc.queuingDelay)
```

Comment:
A long delay here isn't really what we want. It'll cause large queue contents and can queue the same item multiple times. If we're working at the granularity of hours, we should just trigger a full resync once an hour or so